.docs/content/2.guide/1.how-to/how-to-use-htc-mock.md

```diff
@@ -0,0 +1,72 @@
+# How to use HTC Mock?
+
+HTC Mock is a test tool for ArmoniK. It is used to generate tasks that will be runned by ArmoniK. It provides a lot of options to personalize the tasks.
+
+HTC Mock is used in CI to test ArmoniK automatically but it can also be used in development to generate realistic tasks.
+
+::alert{type="info"}
+To populate the database, you can also use some scripts to populate database. See [Populate database?](../populate-database.md)
+::
+
+## Use it
+
+::alert{type="warning"}
+HTC Mock is a development or test tool. It is not intended to be used in production.
+::
+
+To use HTC Mock, you need to swap out the ArmoniK worker in `/infrastructure/quick-deploy/localhost/armonik/parameters.tfvars` with the HTC Mock worker.
+
+```diff [parameters.tfvars]
+    worker = [
+      {
+        name              = "worker"
+-       image             = "dockerhubaneo/armonik_worker_dll"
++       image             = "dockerhubaneo/armonik_core_htcmock_test_worker"
+        tag               = "0.12.1"
+    ]
+```
+
+Then, you can update the tag version. You can verify the [latest of Core on GitHub](https://github.com/aneoconsulting/ArmoniK.Core/release/latest) in order to use it.
+
+:warning: `dockerhubaneo/armonik_control` `dockerhubaneo/armonik_pollingagent` and `dockerhubaneo/armonik_core_htcmock_test_worker` must have the **exact** same tag version. It's a necessity to be sure that they can talk each other using the same API.
+
+Then, you can deploy ArmoniK as usual.
+
+```bash [shell]
+make deploy-armonik
+```
+
+Then you can start the HTC mock client container.
+
+```bash [shell]
+docker run --rm \
+            -e HtcMock__NTasks=100 \
+            -e HtcMock__TotalCalculationTime=00:00:00.100 \
+            -e HtcMock__DataSize=1 \
+            -e HtcMock__MemorySize=1 \
+            -e HtcMock__EnableFastCompute=true \
+            -e HtcMock__SubTasksLevels=1 \
+            -e HtcMock__Partition=htcmock \
```

Comment:
Concerning the partition, if you do what you described in this doc: changing the worker to be htcmock, you will have an htcmock worker but in the default partition. In that case you don't need to specify the partition. (and specifying it to htcmock will not work I think)

Reply:
You have different possibilities : 

1. make the deployment all in one : in that case a partition htcmock with worker htcmock is deployed and you don't need to change anything in the code but think about changing the partition to htcmock when doing the test
2. make the splitted deployment: either you do what you described in this doc changing the worker to htcmock in default partition OR add a partition htcmock with worker htcmock. @YKharouni do you confirm?

Reply:
I am not sure if we need to  present all these possibilities in the doc

Reply:
About the "all-in-one" deployment, there is no documentation about it so we can't talk about it for the moment.

About the name of the partition, maybe, we need to use the default one. Need to try these cases.

Reply:
You should mention it